datastore/datastore/go.mod

```diff
@@ -6,6 +6,7 @@ require google.golang.org/grpc v1.64.0
 
 require (
 	github.com/cridenour/go-postgis v1.0.0
+	github.com/golang/protobuf v1.5.4
 	google.golang.org/protobuf v1.34.1
```

Comment:
@jo-asplin-met-no

Reply:
I'll try to remove it (I think I needed it for my dev env).

Reply:
The problem is that I have a too old protoc (3.12.4) which generates this line in `datastore.pb.go`:
```
timestamp "github.com/golang/protobuf/ptypes/timestamp"
```
instead of this:
```
timestamppb "google.golang.org/protobuf/types/known/timestamppb"
```
which in turn causes `go mod tidy` to add `github.com/golang/protobuf` to `go.mod` in my dev environment. And I have to use go mod tidy, otherwise vscode gets unhappy.

So I'll try to upgrade protoc to something newer (currently it's v3.21.12 that gets installed in the docker image built from `datastore/datastore/Dockerfile`).